pepper-apis/src/main/java/org/broadinstitute/ddp/client/GoogleRecaptchaVerifyClient.java

```diff
@@ -0,0 +1,71 @@
+package org.broadinstitute.ddp.client;
+
+import java.io.IOException;
+
+import com.google.gson.Gson;
+import org.apache.http.HttpResponse;
+import org.apache.http.client.fluent.Form;
+import org.apache.http.client.fluent.Request;
+import org.apache.http.client.fluent.Response;
+import org.apache.http.util.EntityUtils;
+import org.broadinstitute.ddp.exception.DDPException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Client that will verify user reCaptcha responses against Google's reCaptcha verify service
+ * Supports reCaptcha V2 and V3. Which version is being used is based on configuration associated with
+ * site secret key provided.
+ * More info @see <a href="https://developers.google.com/recaptcha">Google reCaptcha</a>
+ */
+public class GoogleRecaptchaVerifyClient {
+    private static final Logger LOG = LoggerFactory.getLogger(GoogleRecaptchaVerifyClient.class);
+    private static final String VERIFY_SERVICE_URL = "https://www.google.com/recaptcha/api/siteverify";
+    private final String secret;
```

Comment:
I imagine the secret will be study-specific, so we'll need to add a place to store and associate it to a study? And every time we want to verify recaptcha, we'll `new` up a client that a particular study/secret?

Reply:
Objective here was to provide a "library" level functionality to check Captcha. How it gets used was TBD.